packages/svelte/src/internal/client/reactivity/sources.js

```diff
@@ -189,6 +189,7 @@ function mark_reactions(signal, status) {
 
 	var runes = is_runes();
 	var length = reactions.length;
+	var to_maybe_dirty = (status & MAYBE_DIRTY) !== 0;
```

Comment:
storing this feels like overkill when we only need to do `status === MAYBE_DIRTY` below (no need for bitwise shenanigans)

```suggestion
```